modules/globals.d.ts

```diff
@@ -0,0 +1,12 @@
+import type { FetchMockStatic } from 'fetch-mock';
+
+declare global {
+  declare var iD: typeof import('.');
+  declare var d3: import('d3');
```

Comment:
```suggestion
  declare var d3: typeof import('d3');
```

Probably like this?

Reply:
done, and I just resolved the git conflicts